packages/tools/src/stateManagement/annotation/annotationVisibility.ts

```diff
@@ -73,7 +73,7 @@ function isAnnotationVisible(annotationUID: string): boolean | undefined {
 /**
  * Get the number of hidden annotation objects in the global set of hidden annotation
  * objects uids.
- * @returns {number} The number of hidden annotation objects.
+ * @returns The number of hidden annotation objects.
  *
  */
 function getAnnotationUIDsHiddenCount(): number {
```

Comment:
Also this doesn't make sense it has UID in the name, can we have `getNumberOfHiddenAnnotations` and any one call .length on it